.github/workflows/CI.yaml

```diff
@@ -75,39 +75,5 @@ jobs:
         file: ./coverage.xml
         flags: unittests
         name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}
+        fail_ci_if_error: True
 
-
-    # Another environment for the the highest versions
```

Comment:
Each job runs though the steps once, you don't need to do multiple set of steps, instead we have the extra job added to `include` that will run minimum tests.